### PR TITLE
Add timestamp option to solve issue #90

### PR DESCRIPTION
**Pull request by jsumners**

### CODE REVIEW

File: pino.js [hunk 1]
```diff
@@ -245,12 +252,16 @@ Pino.prototype.asJson = function asJson (obj, msg, num) {
...
+}
+
 function getTime () {
-  return Date.now()
+  return flatstr(',"time":' + Date.now())
 }
 
 function getSlowTime () {
-  return '"' + (new Date()).toISOString() + '"'
+  return flatstr(',"time":"' + (new Date()).toISOString() + '"')
```
Comment (mcollina) [inline pino.js]: `flatstr` is probably slowing things down here, because we are already doing it later on.

File: pino.js [hunk 2]
```diff
@@ -130,6 +132,11 @@ function Pino (level, stream, serializers, stringify, end, name, slowtime, chind
   this._baseLog = flatstr(baseLog +
     (this.name === undefined ? '' : '"name":' + stringify(this.name) + ','))
 
+  if (this.timestamp === false) {
+    this.time = getNoTime
+    return
```
Comment (mcollina) [inline pino.js]: Why are you using `return` here? It seems not correct to me

File: pino.js [hunk 3]
```diff
@@ -115,12 +116,13 @@ Object.defineProperty(pino, 'levels', {
   enumerable: true
 })
 
-function Pino (level, stream, serializers, stringify, end, name, slowtime, chindings, cache, formatOpts) {
+function Pino (level, stream, serializers, stringify, end, name, timestamp, slowtime, chindings, cache, formatOpts) {
```
Comment (mcollina) [inline pino.js]: can we avoid to have both `timestamp` and `slowtime`?

File: pino.js (hunk 1 above)
Comment (jsumners) [inline pino.js]: The time string was being flattened in `lscache` so I opted to do here as well. My benchmarks aren't showing any real hit, but I can remove it.

File: pino.js (hunk 2 above)
Comment (jsumners) [inline pino.js]: If `timestamp` is `false` then there is no reason to check for the `slowtime` option. This `return` short circuits that check.

File: pino.js (hunk 3 above)
Comment (jsumners) [inline pino.js]: They are different features. How do you propose reducing the verbosity? I would use an object for a single parameter to this function, but I imagine it's done this way for speed.

File: pino.js (hunk 1 above)
Comment (jsumners) [inline pino.js]: I updated it, but for some reason this comment isn't getting flagged as being on an out-of-date commit.

File: pino.js (hunk 2 above)
Comment (mcollina) [inline pino.js]: I prefer to have it as the first check of the if-else right afterwards.

Comment (jsumners) [inline pino.js]: It is amended to be an `if/else if/else`.

**Comment by mcollina** [comment]
Perfect.

**Pull request merged**